src/dataset/actions.ts

```diff
@@ -963,8 +963,9 @@ export async function saveFile(doc: vscode.TextDocument, datasetProvider: IZoweT
             if (node) {
                 node.setEtag(uploadResponse.apiResponse[0].etag);
             }
-        } else if (!uploadResponse.success && uploadResponse.commandResponse.includes(
-            localize("saveFile.error.ZosmfEtagMismatchError", "Rest API failure with HTTP(S) status 412"))) {
+        } else if (!uploadResponse.success && uploadResponse.commandResponse.includes("Rest API failure with HTTP(S) status 412")) {
```

Comment:
This is message is consistent even if there's a change in language, right? I mean. It is an error message from the API so it should be.

Reply:
And it is always going to be 412?

Reply:
Yes in both cases 😅 That's why I removed the localization. It has nothing to do with the error message that z/OSMF generates.